Gradle_Build_Cache_Deep_Dive/demos/cache_miss_timestamp/settings.gradle.kts

```diff
@@ -7,11 +7,12 @@
  * in the user manual at https://docs.gradle.org/7.4.2/userguide/multi_project_builds.html
  */
 plugins {
-    id("com.gradle.enterprise") version "3.8"
+    id("com.gradle.enterprise") version "3.16.2"
 }
 gradleEnterprise {
+    server = "https://dpeuniversity-develocity.gradle.com"
     buildScan {
-        server = "https://enterprise-training.gradle.com"
+        publishAlways()
```

Comment:
I wasn't sure if we wanted to always publish Build Scans or not for these. It originally did not do so.  But I don't know why we wouldn't at this point.